update_changelog.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import datetime
+import glob
+import os
+import sys
+
+import yaml
+
+SKIPLINES = 1
+SECTIONS = [
```

Comment:
Just a note - I believe "server" was added as a section in this release
```suggestion
SECTIONS = [
    "server",
```